pkg/tests/main_test.go

```diff
@@ -21,19 +21,20 @@ import (
 	"os"
 	"os/exec"
 	"path/filepath"
+	"runtime"
+	"strings"
 	"testing"
 
 	"github.com/pulumi/pulumi/pkg/v3/testing/integration"
-	"runtime"
 )
 
-func TestMain(m *testing.M) {
+func TestMains(t *testing.T) {
```

Comment:
These helpers are here what usually Make is for like making sure that build targets are available before running tests. Looks like this change disables them. Is that intentional?

Reply:
Good catch, thanks! I was experimenting with these to see where they are used but didn't mean to commit. 🙌